.travis.yml

```diff
@@ -7,6 +7,7 @@ go:
   - '1.11'
   - '1.12'
   - '1.13'
+  - '1.14'
```

Comment:
Makes sense to add `1.15` as well, IMO

Reply:
When I opened this PR I did not know whether it would fail or succeed with Go 1.14; I would open a new PR to add support for Go 1.15